erigon-lib/state/files_item.go

```diff
@@ -366,17 +366,6 @@ func (files visibleFiles) LatestMergedRange() MergeRange {
 	return MergeRange{}
 }
 
-func (files visibleFiles) MergedRanges() []MergeRange {
```

Comment:
i needed this method for commitment rebuilt. Anyway rebuilt will be revisited when parallel trie comes out, so do not care.